src/analysis/function_parameters.rs

```diff
@@ -297,7 +297,11 @@ pub fn analyze(
             user_data_index: par.closure,
             destroy_index: par.destroy,
             try_from_glib: try_from_glib.clone(),
-            move_: configured_parameters.iter().any(|p| p.move_),
+            move_: if transfer == Transfer::Full && par.direction.is_in() {
+                true
+            } else {
+                configured_parameters.iter().any(|p| p.move_)
+            },
```

Comment:
```suggestion
            move_: configured_parameters.iter().find_map(|p| p.move).unwrap_or(if transfer == Transfer::Full && par.direction.is_in());
```

Otherwise you can't disable it explictly via the configuration. Also means that `p.move` needs to become `Option<bool>`

Reply:
ah right, let me fix that.